src/main/java/StringCalculatorStep1.java

```diff
@@ -5,7 +5,7 @@ public static final int Add(final String numbers) {
         int returnValue = 0;
         String[] numbersArray = numbers.split(",");
         if (numbersArray.length > 2) {
```

Comment:
Consider using a more robust delimiter for splitting the input string, such as regex to handle various types of delimiters (not just comma). This will improve flexibility and maintainability of the code.